pkg/download/bucket/bucket.go

```diff
@@ -75,24 +75,34 @@ func (d *Downloader) Download(projectName string, _ ...config.BucketType) (v2.Co
 
 func (d *Downloader) convertAllObjects(projectName string, objects [][]byte) []config.Config {
 	result := make([]config.Config, 0, len(objects))
+
+	lg := log.WithFields(field.Type("bucket"))
+
 	for _, o := range objects {
 
 		c, err := convertObject(o, projectName)
 		if err != nil {
 			if errors.As(err, &skipErr{}) {
-				log.WithFields(field.Type("bucket")).
-					Debug("Skipping bucket: %s", err.Error())
+				lg.Debug("Skipping bucket: %s", err.Error())
 			} else {
-				log.WithFields(field.Type("bucket"), field.Error(err)).
-					Error("Failed to decode API response objects for bucket resource: %v", err)
+				lg.WithFields(field.Error(err)).Error("Failed to decode API response objects for bucket resource: %v", err)
 			}
 
 			continue
 		}
 		result = append(result, c)
 	}
 
-	log.Info("Downloaded %d Grail buckets", len(result))
+	lg = lg.WithFields(field.F("configsDownloaded", len(result)))
+	switch len(objects) {
+	case 0:
+		// Info on purpose. Most types have a lot of objects, so skipping printing 'not found' in the default case makes sense. Here it's kept on purpose as bucket is only one type.
```

Comment:
This makes me wonder when we should log this as debug and when as info. 
From a user perspective I think it's less dependent on the amount of types there are, but rather if I had requested the type. 

E.g. if I wanted to download a specific setting, I'd probably find an "info No settings found" helpful. 
If I downloaded everything, I don't think I'd need different levels of information for buckets or other types.

Just some thoughts to discuss, and most certainly not a change to do as part of this PR!

Reply:
I had the same thoughts as you're having. If I want to download the schema 'x', but no configs are available, I might want to see a message. For now opted to go this way, as if I'm not sure if it works, users can still enable the debug mode. 

Imo, when we rewrite the download code at some point, we really should include that use-case. However, we would need to restructure our download code immensely to handle that, so currently it's out of scope.